CatAnalyzer/scripts/catGetDatasetInfo

```diff
@@ -44,6 +45,7 @@ for l in csv[1:]:
                'xsec':float(xsec),
                'nevt':int(nevt),
                'lumi':float(lumi),
+               'colour':int(colour),               
```

Comment:
i think ints are more flexible even thou they are not easy to translate for humans